src/mono/mono/mini/CMakeLists.txt

```diff
@@ -304,7 +304,6 @@ endif()
 if(NOT DISABLE_SHARED_LIBS)
   add_library(monosgen SHARED $<TARGET_OBJECTS:monosgen-objects>)
   set_target_properties(monosgen PROPERTIES OUTPUT_NAME monosgen-2.0)
-  set_target_properties(monosgen PROPERTIES LIBRARY_OUTPUT_DIRECTORY .libs)
   target_link_libraries(monosgen ${OS_LIBS} ${ICONV_LIB} ${LLVM_LIBS} ${ICU_LIBS})
```

Comment:
Not sure why this was here in the first place.

Reply:
it replicated what the automake-based build did, outputting libraries into .libs